packages/web-twig/src/Resources/components/Header/_abstracts/Link.twig

```diff
@@ -0,0 +1,31 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _href = props.href | default('#') -%}
+{%- set _isCurrent = props.isCurrent | default(false) -%}
+{%- set _onClick = props.onClick | default(null) -%}
+{%- set _rootClass = props.rootClass -%}
+{%- set _target = props.target | default(null) -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ _rootClass -%}
+{%- set _rootCurrentClassName = _isCurrent ? _spiritClassPrefix ~ _rootClass ~ '--current' : null -%}
```

Comment:
This could be uniform across components—either composed from parts, or just the final class name.

Reply:
No, this is the abstract component.